src/current/_includes/releases/v23.2/v23.2.0-alpha1.md

```diff
@@ -0,0 +1,928 @@
+- SQL statements that must clean up intents from many different previously abandoned transactions now do so moderately more efficiently. [#103265][#103265]
+- The optimizer can now avoid a grouping stage in more cases when de-duplicating the input to an [`UPSERT`]({% link v23.2/upsert.md %}) or `INSERT ... ON CONFLICT` statement. [#105206][#105206]
+- The optimizer can now eliminate joins in more cases. [#105214][#105214]
+- CockroachDB now improves the time to disk space reclamation when deleting rows. Previously, in scenarios where rows had large variations in row size, it was possible for disk space to not be reclaimed after MVCC garbage collection deleted the rows. [#104539][#104539]
+- CockroachDB now has improved disk space reclamation heuristics, making disk space reclamation more timely. [#106177][#106177]
+- `bool_and` and `bool_or` aggregates will now scale linearly instead of quadratically when used as a window function with a non-shrinking window, [#106477][#106477]
+- CockroachDB now has reduced lock contention on `ssmemstorage.RecordStatement`. This is useful for workloads that execute the same statement concurrently on the same SQL instance. [#106860][#106860]
+- The optimizer now produces more efficient query plans in some cases for queries with  subqueries and user-defined functions. [#107133][#107133]
+- The default Raft entry cache size has been increased from 16 MB to 1/256 of system memory with a minimum of 32 MB, divided evenly between all stores. This can be configured using the `COCKROACH_RAFT_ENTRY_CACHE_SIZE` environment variable. [#107424][#107424]
+- CockroachDB now automatically collects table statistics on the `system.jobs` table, which will enable the optimizer to produce better query plans for internal queries that access the `system.jobs` table. This may result in better performance of the system. [#108139][#108139]
+- The impact of high concurrency blind writes to the same key on goroutine scheduling latency was reduced. [#109349][#109349]
+- [Changefeeds]({% link v23.2/change-data-capture-overview.md %}) to Webhook or Pub/Sub endpoints now support much higher throughput [#109351][#109351]
+- This release improved the cost of resolving a user-defined enum type that has many values. [#109394][#109394]
+- Queries that compare collated strings now use less memory and may execute faster. [#110066][#110066]
+- Added a scheduler based rangefeed processor which improves rangefeed and changefeed performance for very large tables. The new processor is disabled by default, but can be enabled by setting `kv.rangefeed.scheduler.enabled` cluster setting to `true`. [#107553][#107553]
+- This release disables `sql.defaults.zigzag_join.enabled` by default. [#110214][#110214]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v23-2-0-alpha1-contributors">Contributors</h3>
+
+This release includes 3116 merged PRs by 122 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
I would delete this section starting at "We would like to thank ..." . I know at least some of these folks are CRL.  This happens b/c they have a personal email in their `.git/config` or similar